wgsl/index.bs

```diff
@@ -7259,6 +7260,7 @@ value with the same sign.
   <tr><td>`modf(x)`<td>Correctly rounded
   <tr><td>`normalize(x)`<td>Inherited from `x / length(x)`
   <tr><td>`pow(x, y)`<td>Inherited from `exp2(y * log2(x))`
+  <tr><td>`radians(x)`<td>Inherited from `x * 0.017453292519943295474`
```

Comment:
is this going to be any faster than the user doing the multiplication?

Reply:
No.  It's user convenience offered in HLSL and GLSL.

Reply:
No.  It's user convenience offered in HLSL and GLSL